tron/bin/check_tron_jobs.py

```diff
@@ -337,6 +338,10 @@ def sort_runs_by_interval(job_content, interval='day', until=None):
                 time.mktime(_timestamp_to_timeobj(run['run_time']))
                 for run in job_content['runs']
             ])
+            earliest_run_time = max(
+                earliest_run_time,
```

Comment:
This logic is about 12 lines long. What do you think a function called 'calculate_earliest_run_time' that took `job_content` and `until` ?